.github/workflows/upstream-compat.yaml

```diff
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches-ignore:
```

Comment:
Do we need to exclude tags here :thinking: ?

Reply:
good point. I'll change it to use `branches` instead of `branches-ignore` and use `!main`/`!for-upstream/**` - this way it's clear we only want to run for branches except those matching and never for tags

Reply:
hm this does not seem to have triggered the workflow though:
https://github.com/anexia-it/machine-controller/releases/tag/test-upstream-workflow-tag

Reply:
another look at this would be great